src/main/kotlin/no/nav/familie/ef/iverksett/iverksetting/domene/Simulering.kt

```diff
@@ -13,6 +13,6 @@ fun Iverksett.tilSimulering() = Simulering(
                 eksternFagsakId = this.fagsak.eksternId,
                 personIdent = this.søker.personIdent,
                 behandlingId = this.behandling.behandlingId,
-                vedtaksdato = this.vedtak.vedtaksdato
+                vedtaksdato = this.vedtak.vedtakstidspunkt!!.toLocalDate()
```

Comment:
tenker at det er "enklere" å finne frem hvis vi bruker `vedtakstidspunkt?.toLocalDate() ?: throw Feil("Vedtakstidspunkt er ikke satt")` 

Har en kjepphest mot `!!` på generelt grunnlag i produksjonskode, men det er en annen sak 😂

Reply:
Det er feil i kontrakten at vedtakstidspunkt er nullable nå, fordi ef-sak alltid sender med vedtakstidspunkt etter å ha implementert den nye kontrakten. Derfor tillot jeg meg å bruke `!!` i stedet for å håndtere noe som ikke skal skje med `?`.  Har planer om å merge https://github.com/navikt/familie-kontrakter/pull/441 så fort jeg har fått merget denne PR'n

Reply:
Kan like gjerne merge denne: https://github.com/navikt/familie-kontrakter/pull/441
også kan jeg fjerne `!!` i denne PR'n, slik at den koden aldri kommer i prod.

Reply:
Som Charlie nevner her/annen kommentar så unngår vi `!!` hvis domene-objektet i iverksett er required, kontrakter har ikke veldig mye å si :)